src/condor_daemon_client/daemon.cpp

```diff
@@ -1584,7 +1584,7 @@ Daemon::findCmDaemon( const char* cm_name )
 			return false;
 		}
 		sinful.setHost(saddr.to_ip_string().c_str());
-		sinful.setAlias(fqdn.c_str());
+		sinful.setAlias(host);
```

Comment:
```suggestion
```
Deleting this call matches the logic in getDaemonInfo() when hostname:port is given. The call to New_alias() here should cause the given alias to be added to the sinful that's set in New_addr().